lib/Conversion/TritonGPUToLLVM/TypeConverter.cpp

```diff
@@ -7,6 +7,7 @@ using namespace mlir;
 using namespace mlir::triton;
 
 using ::mlir::triton::gpu::AMDMfmaEncodingAttr;
+using ::mlir::triton::gpu::AMDWmmaEncodingAttr;
```

Comment:
Is there a reason why we added this if it's not being used ?

Reply:
I turned back this changes https://github.com/openai/triton/pull/3308/files#diff-928f898f93db4afa3551c43a4bdc36b127c0c866f76ac7cdc90bae10116972e9R89 .

But after this PR is merged https://github.com/openai/triton/pull/3357
I'll remove it and take into account your comment.